sdks/python/apache_beam/ml/inference/run_inference.py

```diff
@@ -0,0 +1,43 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import apache_beam as beam
+from apache_beam.ml.inference.model import (
+    PytorchModel, RunInferenceModel, SklearnModel)
```

Comment:
I think 3 import statements should be written individually to remove the Lint error.

Reply:
I have a habit of this due to experiences in other, older Python repos 😆

Reply:
Ideally we should not have to import specific implementations here, just RunInferenceModel.